14-01-date-input/14-1-EN-date-input.sbv

```diff
@@ -249,7 +248,7 @@ exactly 24 hours ago.
 However, if we need yesterday morning
 
 0:04:00.220,0:04:01.640
-it's easy to make
+it's easy to make it
```

Comment:
Do we need it here? Because together with the next sentence it reads:
"Its easy to make it"
"Let's make it ..."
Does it feel better? Im not sure here, what do you think?

Reply:
anyway question lasts. to make what? like in the middle of thought. english doesn't suppose smth like that. there is need  in ending. because the next one isn't an extension. it's new sentence . actually as this is not a doc both are fine.